packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -129,6 +132,27 @@ const FIELD_MIN_WIDTH_MONTH_YEAR = 167;
  * />
  * ```
  */
+
+/* [WB-1655] Update with media query tokens */
+const xsMin = "520px";
+
+const screenSizes = {
+    small: `@media (max-width: ${xsMin})`,
+};
+
+const defaultStyles = StyleSheet.create({
+    wrapper: {
+        flexDirection: "row",
+        [screenSizes.small]: {
```

Comment:
**suggestion:** We can add a story for the BirthdayPicker that will default to a small viewport so that we have a visual regression test with Chromatic for this scenario! 

This can be done by adding a story like this following the [Storybook viewport docs](https://storybook.js.org/docs/essentials/viewport#defining-the-viewport-for-a-story):
```ts
export const Responsive: StoryComponentType = {
    parameters: {
        viewport: {
            defaultViewport: "small",
        },
    },
};
```

I couldn't find any examples in WB where we default to a small viewport, though the [ModalDialog stories](https://github.com/Khan/wonder-blocks/blob/f89fc029184e2319e643d0001ea95d334ca5e2cf/__docs__/wonder-blocks-modal/modal-dialog.stories.tsx#L84) have a default viewport set to a `desktop`. It also has `customViewports` defined, which I think we don't need because we have some [WB viewports defined in Storybook](https://github.com/Khan/wonder-blocks/blob/f89fc029184e2319e643d0001ea95d334ca5e2cf/.storybook/preview.tsx#L14-L43) globally!

Reply:
Great idea. Thanks Bea!

Reply:
Added a test!